lib/ood_appkit/urls/editor.rb

```diff
@@ -4,7 +4,7 @@ module Urls
     class Editor < Url
       # @param (see Url#initialize)
       # @param edit_url [#to_s] the URL used to request the file editor api
-      def initialize(edit_url: '/edit', template: '{/url*}{+path}', **kwargs)
+      def initialize(edit_url: '/edit', template: '{/url*}{/fs}{+path}', **kwargs)
```

Comment:
I'm unfamiliar with Ruby's URI templates. Apparently there's an RFC around the same. In any case, I want to be sure the `*` in `{/url*}` and the `+` in `{+path}` don't have odd side affects with this literal.

Reply:
As far as I know it shouldn't cause any side effects, though now that I think more about it we might actually want `{/url*}/{+fs}{+path}` here instead.
The `*` allows us to pass an array, which is what `parse_url_segments` gives us, e.g.
```
3.0.2 :023 > Addressable::Template.new("{/url*}{/fs}{+path}").expand(url: ["foo", "bar"], fs: ["foo", "bar"]).to_s
 => "/foo/bar/foo,bar" 
```
The `+` allows characters that would otherwise be encoded, e.g.
```
3.0.2 :027 > Addressable::Template.new("{/url*}{/fs}{+path}").expand(url: "foo", fs: "bar?", path: "/baz?").to_s
 => "/foo/bar%3F/baz?"
```
Since the Rclone remotes names can have spaces in them this might be something we need on `fs`, though we'd have to move the `/` outside as I didn't seem to be able to combine them.

Reply:
This library has a testing suite, we should probably add tests to at least be sure whatever we choose behaves as expected.

https://github.com/OSC/ood_appkit/blob/ed34677df08a50d7bea957764ae09f9b2888d990/test/ood_appkit_test.rb#L21

Reply:
Sure, I'll add tests.

Reply:
Added the tests.
For the possible Rclone remote names there is no difference between `/{+fs}` and `{/fs}`, so I kept the current format. Spaces are encoded as `%20`, this is fine, and I've verified that it works as expected in OOD.

Reply:
Cool, I'm on this today so we should have movement on it.